modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/src/main/resources/site-initializer/fragments/group/testray/fragments/login/fragment.json

```diff
@@ -1,5 +1,5 @@
 {
-	"configurationPath": "index.json",
+	"configurationPath": "configuration.json",
```

Comment:
@samuelleolopes this is right? do we have this file?

Reply:
It will not be necessary, it will remain the file 'index.json'